src/TraceEvent/Ctf/CtfTracing.Tests/CtfTracing.Tests.csproj

```diff
@@ -36,19 +36,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework" />
   </ItemGroup>
-  <Choose>
```

Comment:
> First I would like to understand if we can support BOTH without too much trouble. If this takes less than a couple hours, I would like to do that.

It is easy for me to account for both. I'd like the opportunity during the normal work week (not taking up a weekend) to have a brief phone call (say 15 minutes) and make my case for the change as I proposed, and if I do not prove to be convincing in your mind then we'll go with a hybrid approach. :smile: